int8 data_type  # enum of sample type of requested data, defined in gps_pb2
float64[] data
int32[] shape
